app/models/list_entry.rb

```diff
@@ -0,0 +1,4 @@
+class ListEntry < ActiveRecord::Base
+  belongs_to :list
+  belongs_to :book
```

Comment:
Could be nice to have the "has_many" association on Books model too. The same "has_many :list_entries" of List.